### DISCUSSION TITLE
Problem with Loading Global CSS in Docker APP

### DISCUSSION THREAD

**Discussion by sanderkooger** [vercel/next.js#14337] [Help]
# Bug report
Same as #12226

We are building an app that runs Nextjs, MUI and react-multi-carrousel, The latter requires a CSS import which we handle in our _app.css This app needs to be built as a dockerimage. Local builds work, But as soon as we build with docker the build fails on npm run build. 

## Describe the bug
Building on a machine works, 
Buildign in a docker container generates an error 

![image](https://user-images.githubusercontent.com/19397354/85070082-7a766d80-b1b5-11ea-9b1f-4e3cbd025ba7.png)

## To Reproduce

1 - Create _app.js with global css.
2 - Create a DockerFile with Ubuntu as OS.
3 - Build next in DockerFIle.
4 - Problem with global css should occur.

or pull the nextjs example with react-multi-slider and add dockerfile on the bottom of this post.

## Expected behavior

Successful build like on a normal machine

## Screenshots
![image](https://user-images.githubusercontent.com/19397354/85070348-ee187a80-b1b5-11ea-8752-d0022a552d9c.png)

## System information

- Alpine node 12
- Browser (if applies) [e.g. chrome, safari]
- Version of Next.js: [e.g. 9.4.4]
- Version of Node.js: [e.g. 12]

## Additional context
```
FROM mhart/alpine-node:12 AS builder
RUN addgroup -S www-data && adduser -S -g www-data www-data && mkdir app && chown www-data:www-data app
USER www-data
WORKDIR /app
COPY package*.json .
RUN yarn install
COPY . /app
RUN yarn build && yarn --production

FROM mhart/alpine-node:12
RUN addgroup -S www-data && adduser -S -g www-data www-data && mkdir app && chown www-data:www-data app
WORKDIR /app
COPY --from=builder /app .
EXPOSE 3000
USER www-data
CMD ["yarn", "start"]
```

Another issue referenced to this issue as well but its closed.  #12226

Access to our repo is possible if needed.

**Comment by zilkey** [comment]
I had the same problem. This dockerfile seemed to work, FWIW:

```
FROM node:alpine

RUN id -u node
WORKDIR /app

COPY package.json .
COPY yarn.lock .
RUN yarn install --prod

COPY pages pages
COPY public public
COPY src src
RUN yarn build

EXPOSE 3000
USER node
ENTRYPOINT ["npm", "start"]
```
